React/tic-tac/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import style from './style.scss';
+
+class LoginForm extends Component {
+  state = {
+    loggedIn: false
+  };
+
+  loginHandle = () => {
+    this.setState(() => ({ loggedIn: true }));
+  };
+
+  render() {
+    const status = this.state.loggedIn ? (
+      <Redirect to="/Game" />
```

Comment:
This is OK, however, this is not the `LoginForm` responsibility. You have a `Router` component where you can handle redirects when the user is or is not logged in.